index.js

```diff
@@ -162,50 +158,50 @@ utils.prototype.getWeekArray = function getWeekArray(date) {
   });
 
   return nestedWeeks;
-}
+};
 
 utils.prototype.getYearRange = function getYearRange(start, end) {
-  const startDate = parse(start);
-  const endDate = parse(end);
-  const years = [];
+  var startDate = parse(start);
+  var endDate = parse(end);
+  var years = [];
 
-  let current = startDate;
+  var current = startDate;
   while (current.isBefore(endDate)) {
     years.push(current);
     current = current.clone().add(1, 'jYear');
   }
 
   return years;
-}
+};
 
 utils.prototype.getCalendarHeaderText = function getCalendarHeaderText(date) {
   return date.format('jMMMM jYYYY');
-}
+};
 
 utils.prototype.getYearText = function getYearText(date) {
   return date.format('jYYYY');
-}
+};
 
 utils.prototype.getDatePickerHeaderText = function getDatePickerHeaderText(date) {
   return date.format('ddd, jMMM jD');
-}
+};
 
 utils.prototype.getDateTimePickerHeaderText = function getDateTimePickerHeaderText(date) {
   return date.format('jMMM jD');
-}
+};
 
 utils.prototype.getDayText = function getDayText(date) {
   return date.format('jD');
-}
+};
 
 utils.prototype.getHourText = function getHourText(date, ampm) {
   return date.format(ampm ? 'hh' : 'HH');
-}
+};
 
 utils.prototype.getMinuteText = function getMinuteText(date) {
   return date.format('mm');
-}
+};
 
-utils['default'] = utils;
+utils.default = utils;
```

Comment:
Please revert this. `default` is a contextual keyword and might cause problems with some browsers. Please revert this particular change.